tests/glimmer/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1501,7 +1538,9 @@ singleQuote: true
     )
     hashPair=(does not need a line break due to being under 80 chars long)
   )
-}}{{/block}}
+}}
+
+{{/block}}
```

Comment:
Looks ugly

Reply:
fair enough, I'll address this now

Reply:
While fixing this, these were also fixed:

**before:**

```hbs
{{#block}}{{#block}}hello{{/block}}{{/block}}
{{#block}}hello{{/block}}
```

**after:**

```hbs
{{#block}}
  {{#block}}
    hello
  {{/block}}
{{/block}}
{{#block}}
  hello
{{/block}}
```